addons/web_editor/static/src/scss/web_editor.frontend.scss

```diff
@@ -31,3 +31,21 @@
         }
     }
 }
+
+.o_we_shape {
+    @include o-position-absolute(0,0,0,0);
+    position: absolute !important;
+    background-repeat: no-repeat;
+}
+
+.o_we_flip_x {
+    transform: scale(-1,1);
+}
+
+.o_we_flip_y {
+    transform: scale(1,-1);
+}
+
+.o_we_flip_x.o_we_flip_y {
+    transform: scale(-1,-1);
+}
```

Comment:
No blank lines between related css rules + add a space after `,` inside `scale` (or actually use `scaleX(-1)`, `scaleY(-1)` and `scale(-1)` which would be more explicit)